plugins/docker/docker_

```diff
@@ -0,0 +1,217 @@
+        echo "thinpoolminfreespace.info Thin Pool Minimum Free Space in bytes"
+        ;;
+      esac
+    done
+    ;;
+  storage-percent)
+    echo "graph_title Docker storage in percent"
+    echo "graph_args --upper-limit 100 -l 0"
+    echo "graph_vlabel %"
+    echo "graph_scale no"
+    echo "graph_category docker"
+    echo "graph_info Storage usage in percent"
+    /usr/bin/docker info | while read line; do
+      KEY=$( echo "$line"|cut -f1 -d:|tr ' ' '_' )
+      case ${KEY} in
+      Data_Space_Used)
+        echo "dataspaceused-percent.label Data Space Used"
+        echo "dataspaceused-percent.info Data Space Used in percent"
+        ;;
+      Metadata_Space_Used)
+        echo "metadataspaceused-percent.label Metadata Space Used"
+        echo "metadataspaceused-percent.info Metadata Space Used in percent"
+        ;;
+      esac
+    done
+    ;;
+  esac
+  exit 0
+  ;;
+esac
+
+DATASPACETOTAL=$( /usr/bin/docker info|grep "Data Space Total"|cut -f2 -d:|tr -d 'B'| tr -d ' ')
+METADATASPACETOTAL=$( /usr/bin/docker info|grep "Metadata Space Total"|cut -f2 -d:|tr -d 'B'|tr -d ' ' )
+
+/usr/bin/docker info | while read line; do
+  KEY=$( echo "$line"|cut -f1 -d:|tr ' ' '_' )
+  VAL=$( echo "$line"|cut -f2 -d:|cut -f2 -d' ' )
+  UNT=$( echo "$line"|cut -f2 -d:|cut -f3 -d' '|tr -d 'B'| awk '{print toupper($0)}' )
+  case ${TYPE} in
+  container)
+    case ${KEY} in
+    Containers)
+      echo "containers.value ${VAL}"
+      ;;
+    Running)
+      echo "running.value ${VAL}"
+      ;;
+    Paused)
+      echo "paused.value ${VAL}"
+      ;;
+    Stopped)
+      echo "stopped.value ${VAL}"
+      ;;
+    Images)
+      echo "images.value ${VAL}"
+      ;;
+    esac
+    ;;
+  storage)
+  case ${KEY} in
+    Pool_Blocksize)
+      echo "poolblocksize.value $(numfmt --from=si ${VAL}${UNT} )"
+      ;;
+    Base_Device_Size)
+      echo "basedevsize.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Data_Space_Used)
+      echo "dataspaceused.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Data_Space_Total)
+      echo "dataspacetotal.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Data_Space_Available)
+      echo "dataspaceavail.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Metadata_Space_Used)
+      echo "metadataspaceused.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Metadata_Space_Total)
+      echo "metadataspacetotal.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Metadata_Space_Available)
+      echo "metadataspaceavail.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    Thin_Pool_Minimum_Free_Space)
+      echo "thinpoolminfreespace.value $(numfmt --from=iec ${VAL}${UNT} )"
+      ;;
+    esac
+    ;;
+  storage-percent)
+  case ${KEY} in
+    Data_Space_Used)
+      USED=$(numfmt --from=iec ${VAL}${UNT} )
+      TOTAL=$(numfmt --from=si $DATASPACETOTAL )
+      VALUE=`bc <<< "scale=2;$USED*100/$TOTAL"`
```

Comment:
Maybe you want to use `awk` for this calculation in order to avoid the dependency on `bc`?
```shell
echo "$USED" "$TOTAL" | awk '{ print(($1 * 100) / $2);
```